lib/messagebird/voice_client.rb

```diff
@@ -0,0 +1,27 @@
+require "net/https"
+require "uri"
+require "json"
+require "messagebird/http_client"
+
+module MessageBird
+  class VoiceClient < HttpClient
+    attr_reader :endpoint
+
+    BASE_ENDPOINT = "https://voice.messagebird.com"
+
+    def initialize(access_key)
+      super(access_key)
+      @endpoint = BASE_ENDPOINT
+    end
+
+    def endpoint()
+      @endpoint
```

Comment:
i see in the project is used constant `https://github.com/messagebird/ruby-rest-api/blob/master/lib/messagebird/http_client.rb#L20`